examples/advanced/propagator/src/FairEveRecoTracksExampleEditor.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * FairEveRecoTracksExampleEditor.h
+ *
+ *  Created on: 27 sie 2020
+ *      Author: Daniel Wielanek
+ *		E-mail: daniel.wielanek@gmail.com
+ *		Warsaw University of Technology, Faculty of Physics
+ */
+#ifndef FAIREVERECOTRACKSEXAMPLEEDITOR_H_
+#define FAIREVERECOTRACKSEXAMPLEEDITOR_H_
+
+#include "FairEveCut.h"
+#include "FairEveRecoTracksExample.h"
+
+#include <TGedFrame.h>
+
+class FairEveRecoTracksExampleEditor : public TGedFrame
+{
+    FairEveMinMaxCut *fPtCut;
+    FairEveMinMaxCut *fEtaCut;
+    FairEveMinMaxCut *fEnergyCut;
+    FairEveRecoTracksExample *fRecoTracks;
+    FairEveBoolCut *fDrawMC;
```

Comment:
```suggestion
    std::unique_ptr<FairEveMinMaxCut> fPtCut;
    std::unique_ptr<FairEveMinMaxCut> fEtaCut;
    std::unique_ptr<FairEveMinMaxCut> fEnergyCut;
    FairEveRecoTracksExample *fRecoTracks;
    std::unique_ptr<FairEveBoolCut> fDrawMC;
```

Avoid owning raw pointers if possible. See [R.smart](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#rsmart-smart-pointers) and [R20](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#r20-use-unique_ptr-or-shared_ptr-to-represent-ownership) in particular.